pyiron/atomistics/structure/_visualize.py

```diff
@@ -0,0 +1,669 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import numpy as np
+import warnings
+from matplotlib.colors import rgb2hex
+from pyiron_base import Settings
+from scipy.interpolate import interp1d
+
+__author__ = "Joerg Neugebauer, Sudarsan Surendralal"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Sudarsan Surendralal"
+__email__ = "surendralal@mpie.de"
+__status__ = "production"
+__date__ = "Sep 1, 2017"
+
+s = Settings()
+
+class Visualize:
+    def __init__(self, job):
+        self._job = job
+
+    def plot3d(self, mode='NGLview', **kwargs):
```

Comment:
I guess this docstring is not valid until all the modes take the same arguments. I don't have strong feelings about how to handle this.